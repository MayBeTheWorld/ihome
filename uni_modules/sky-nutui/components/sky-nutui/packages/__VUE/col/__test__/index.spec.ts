import Col from '../index.vue';
import Row from '../../row/index.vue';
import { mount } from '@vue/test-utils';

// 测试用例
test('should render Col correctly and should render gutter correctly', async () => {
  const wrapper = mount({
    components: {
      Row: Row,
      Col: Col
    },
    template: `
      <Row gutter="6">
        <Col span="8">8</Col>
        <Col span="8">8</Col>
        <Col span="8">8</Col>
      </Row>
    `
  });
  expect(wrapper.html()).toMatchSnapshot();
});
