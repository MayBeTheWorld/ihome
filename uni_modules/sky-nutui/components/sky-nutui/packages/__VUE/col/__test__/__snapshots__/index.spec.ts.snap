// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Col correctly and should render gutter correctly 1`] = `
"<view class=\\"nut-row-
              nut-row-justify-start
              nut-row-align-flex-start
              nut-row-flex-nowrap
              nut-row\\">
  <view class=\\"nut-col nut-col-gutter nut-col-8 nut-col-offset-0\\" style=\\"padding-left: 3px; padding-right: 3px;\\">8</view>
  <view class=\\"nut-col nut-col-gutter nut-col-8 nut-col-offset-0\\" style=\\"padding-left: 3px; padding-right: 3px;\\">8</view>
  <view class=\\"nut-col nut-col-gutter nut-col-8 nut-col-offset-0\\" style=\\"padding-left: 3px; padding-right: 3px;\\">8</view>
</view>"
`;
